src/ray/core_worker/store_provider/memory_store/memory_store.cc

```diff
@@ -396,4 +396,16 @@ bool CoreWorkerMemoryStore::Contains(const ObjectID &object_id, bool *in_plasma)
   return false;
 }
 
+MemoryStoreStats CoreWorkerMemoryStore::GetMemoryStoreStatisticalData() {
+  absl::MutexLock lock(&mu_);
+  MemoryStoreStats item;
+  for (const auto &it : objects_) {
+    if (!it.second->IsInPlasmaError()) {
+      item.num_local_objects += 1;
+      item.used_object_store_memory += it.second->GetSize();
```

Comment:
Hi @zzyunzhi, why we count `used_object_store_memory` for the object which is not in plasma?

Reply:
Maybe I misunderstand the `used_object_store_memory`. Does it point to plasma store memory?

Reply:
Yes I believe `used_object_store_memory` only accounts for plasma store memory, sorry for the naming confusion

Reply:
is there a specific need for exposing object storage stats outside plasma in your case?

Reply:
thanks @zzyunzhi for the answer. Just for debug purpose because we see some plasma eviction warning which has been fixed in the latest branch. One more question: does this `if (!it.second->IsInPlasmaError())` means object is not stored in plasma? Should we just count for plasma objects while here for memory store objects?